feedback/lib/src/l18n/translation.dart

```diff
@@ -211,6 +229,7 @@ class GlobalFeedbackLocalizationsDelegate
     const Locale('fr'): const FrFeedbackLocalizations(),
     const Locale('ar'): const ArFeedbackLocalizations(),
     const Locale('ru'): const RuFeedbackLocalizations(),
+    const Locale('sv'): const RuFeedbackLocalizations(),
```

Comment:
```suggestion
    const Locale('sv'): const SvFeedbackLocalizations(),
```

Reply:
🤦 sry about that.. i was trying to run it by using git urls in my pubspec but the feedback_sentry plugin was freaking out about it.  i ended up editing the cached source files and forgot to update this PR

Reply:
No worries :D